src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig

```diff
@@ -0,0 +1,18 @@
+<form id="theme_search" method="post" action="{{ path('search_user_search') }}{# @todo probably an incorrect path #}">
+    <div>
+        <input id="block_search_q" type="search" name="q" size="20" maxlength="255" results="10" autosave="Search" class="theme_search_input" />
+        {% if vars.displaySearchBtn == 1 %}
+        <input class="theme_search_button" type="submit" value="{gt text="Search" domain='zikula'}" />
```

Comment:
and [here](https://github.com/zikula/core/commit/eb4112e0d8741a73e3a2d10118c24698e6c40925)